### DISCUSSION TITLE
什麼???你沒聽過步進馬達

### DISCUSSION THREAD

**Discussion by zhilin93517** [1101-GS4719-C/Python-Community-GS4719-C#2177] [General]
你有聽過步進馬達嗎？你不知道步進馬達在我們生活中的運用嗎？

步進馬達是一種馬達，他可以根據訊號的數量決定要旋轉的角度，一個脈衝訊號可以讓步進馬達前進一步。
步進馬達的內部是多格電磁鐵所組成，透過改變電磁鐵的Ｎ與Ｓ極可以讓步進馬達前進特定的角度，所以生活中的運用是用來控制精準角度，可以讓角度的轉動幅度達到很高的程度。

我有曾經使用步進馬達進行角度的控制，我搭配螺桿，讓物體可以沿著螺桿前進，快速抵達目的地的同時也可以精準控制物體的前進長度，很大程度的幫助了我的使用。我認為大家有空的話都可以來學看看步進馬達的結構，因為步進馬達是很方便的一個小零件，說不定在哪天可以派上用場!

**Comment by zhilin93517** [comment]
如果可以的話可以讓步進馬達搭配 TB6600 使用，TB6600可以根據輸入訊號分配應該要輸入進步進馬達的脈衝，不但增加了順逆時針旋轉的控制，也可以針對轉動角度的精準程度進行控制。此外，還有很多模組可以搭配步進馬達使用，我認為這對於控制可以有很大幅度的改善。

**Comment by Jinyuu17** [comment]
窮學生買不起步進馬達，我都用超級便宜的伺服馬達。可以控制角度，單相對地沒有那麼精準。

**Reply by zhilin93517** [reply]
伺服馬達和步進馬達常常被一起提起，他們各有優缺點，比如伺服馬達的轉速可以提得很高，但步進馬達沒辦法。

**Comment by gigitilto** [comment]
步進馬達確實是一個很好的東西，他可以很精確的控制位置，像是之前我在使用3D列印機以及雷射雕刻機的時候，就有看到步進馬達，如果沒有這個東西的存在，打印的東西就不會那麼完美了。

**Comment by zhilin93517** [comment]
因為連接模組後，步進馬達的線會變得非常多，想要檢查馬達是否可以正常運行但又還沒把馬達的線接好的話，可以先試著對馬達的接收埠使用訊號產生器輸入方波，這樣可以模擬脈衝訊號，直接進行餵步。
